File: JsonDummyServer/app/controllers/ResultObject.java
package controllers;

import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;
import java.util.Map;

public class ResultObject {
	public String id;
	public String name;
	public String description;
	public Boolean flag;
	public Double real;
	public Map<String, String> properties = new HashMap<String, String>();
	public List<Long> values = new ArrayList<Long>();
}

File: PerftestsIOS/PerftestsIOS/ResultObject.m
//
//  ResultObject.m
//  PerftestsIOS
//
//  Created by Administrator on 07.04.13.
//  Copyright (c) 2013 ObjectCode GmbH. All rights reserved.
//

#import "ResultObject.h"

@implementation ResultObject

@synthesize id_;
@synthesize name;
@synthesize description;
@synthesize flag;
@synthesize properties;
@synthesize values;

-(void)parse:(NSDictionary*)json {
    self.id_ = [json objectForKey:@"id"];
    self.name = [json objectForKey:@"name"];
    self.description = [json objectForKey:@"description"];
    self.flag = [json objectForKey:@"flag"];
    self.real = [json objectForKey:@"real"];
    NSDictionary *props = [json objectForKey:@"properties"];
    for(id key in props) {
        id value = [props objectForKey:key];
        [self.properties setValue:value forKey:key];
    }
    self.values = [json objectForKey:@"values"];
}

@end
